### ISSUE TITLE
RavenJS and Sentry blocking

### ISSUE THREAD

**Issue by lightswitch05** [lightswitch05/hosts#58]
RavenJS (cdn.ravenjs.com) and Sentry (sentry.io) is currently blocked by this list.

### Why I added them to the list to start with

 First, I'm not a lawyer, and their privacy policy doesn't make clear distinctions between people who signup for their services and people who are tracked by their services. However, here are the sections from the privacy policy that flagged my attention to add these domains to my list:

> We may also use the data to [...] improve the relevance of our advertising.

> **Improving Advertising Campaigns**. We may use your data to improve our advertising campaigns, primarily in an effort to prevent targeting of impressions via third-party channels when they are not relevant to you.

> We may use and share aggregated non-personal information with third parties for marketing, advertising, and analytics purposes.

### Options provided to have them removed from the list

If any of these three features get implemented, I'll take actions to remove them from the list.

1. [Stop using `sentry.io` as the API domain](https://github.com/StevenBlack/hosts/commit/521777278a3a496c94dda6d363695714fce2f247#r30820781). I don't like blocking this root domain, but its the domain that is used for the API endpoint.
2. [Respect browser DNT settings](https://github.com/StevenBlack/hosts/commit/521777278a3a496c94dda6d363695714fce2f247#r30820962). Not making API requests - or in some cases even loading the library if the browser has the DNT flag enabled 
3. [Support EFF's DNT policy](https://github.com/lightswitch05/hosts/commit/6a256c58cc46b43e5961373c3e5640743c178932#commitcomment-29582245). The EFF has a comprehensive DNT policy that would allow them to continue to collect non personal identifiable information.

### Other conversations.

* https://github.com/lightswitch05/hosts/commit/6a256c58cc46b43e5961373c3e5640743c178932#commitcomment-29581997
* https://github.com/getsentry/sentry-javascript/issues/1421
* https://github.com/StevenBlack/hosts/issues/785
* https://github.com/lightswitch05/hosts/commit/78eba1c684e900845387f8e2cfdcbc770c266289#commitcomment-30290114
* https://github.com/StevenBlack/hosts/issues/881

**Comment by dnmTX** [comment]
![creatingalabel](https://user-images.githubusercontent.com/34774426/51220322-68c2d380-18fa-11e9-908a-8b97256dbb4e.png)

Make it Yellow so even the blind can see it  :smiling_imp:

**Comment by dnmTX** [comment]
Did some research and found that:
`browser.sentry-cdn.com`
`s1.sentry-cdn.com`
are sharing the same IP as `js.sentry-cdn.com`. 
Listing them here for consideration and further research.

**Comment by lightswitch05** [comment]
Thanks, I'll keep them in mind. If I ever see them being used as a tracker, I won't hesitate to add them.

**Comment by lightswitch05** [comment]
Looks like Safari is removing the DNT feature. So the options above about respecting people's DNT flags are no longer an option to have them removed from the list. However, if they wanted to start using a subdomain for the tracking instead of their root domain, then I would still be willing to remove the root block.

**Comment by lightswitch05** [comment]
Why blocking Sentry sometimes breaks websites: https://github.com/getsentry/sentry-docs/pull/814

TLDR: Sentry encourages unsafe usages of their library in their documentation

**Comment by spirillen** [comment]
This is a older thread and by bandwidth is rather limited....

In case you haven't added `sentry-cdn.com` in the tracking list, you can find traces of that here: <https://www.mypdns.org/search/query/eLFQ4Ibkb4AC/#R>

**Comment by lightswitch05** [comment]
Thanks @spirillen - it is currently blocked: https://blocklist-tools.developerdan.com/entries/search?q=sentry-cdn.com
